pkg/authenticator/authenticator.go

```diff
@@ -314,3 +318,25 @@ func decodeFromPEM(PEMBlock []byte, publicCert *x509.Certificate, privateKey cry
 
 	return decodedPEM, nil
 }
+
+func getInjectClientCertError(path string) string {
+	// The log file will not exist in old Conjur versions
+	err := utils.VerifyFileExists(path)
+	if err != nil {
+		log.Warn(log.CAKC001W, path)
+		return ""
+	}
+
+	content, err := ioutil.ReadFile(path)
+	if err != nil {
+		log.Error(log.CAKC034E, path)
+		return ""
+	}
+
+	err = os.Remove(path)
```

Comment:
nit: We should probably log that we're removing this file since it's a destructive op